### PR TITLE
feat: consume wrappers

### PR DESCRIPTION
**Pull request by mJarsater**
Refactored app to use only wrappers.

**Solving issue**  
Fixes: [CDEP-2590](https://tegel.atlassian.net/browse/CDEP-2590)

**This should wait until we release a 1.0 version of the wrappers.**

### CODE REVIEW

**Comment by timrombergjakobsson** [comment]
Getting this in web inspector console. 
Skärmavbild 2023-09-18 kl  10 39 58

**Comment by timrombergjakobsson** [comment]
Maybe not related this one, but when clicking Toggle button.
Skärmavbild 2023-09-18 kl  10 46 12

**Comment by mJarsater** [comment]
> Maybe not related this one, but when clicking Toggle button. Skärmavbild 2023-09-18 kl 10 46 12

Needed to change all references to not use `nativeElement` in the demo page, should be fixed now

File: src/app/components/accordion/accordion.component.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
-import { CUSTOM_ELEMENTS_SCHEMA, Component, Output, EventEmitter } from '@angular/core';
+import {  Component, Output, EventEmitter } from '@angular/core';
+import { TegelModule } from '@scania/tegel-angular';
 
 @Component({
   selector: 'app-accordion',
   templateUrl: './accordion.component.html',
   standalone: true,
-  schemas: [CUSTOM_ELEMENTS_SCHEMA],
```
Comment (alexandereneroth) [inline src/app/components/accordion/accordion.component.ts:7]: We wouldn't have to import it everywhere if it wasn't for standalone: true, right?

Comment (mJarsater) [reply src/app/components/accordion/accordion.component.ts:7]: yes

Comment (timrombergjakobsson) [reply src/app/components/accordion/accordion.component.ts:7]: That is correct!

**Comment by timrombergjakobsson** [comment]
Seems to work nicely! I guess we should create a ticket for these null checks to be handled in Tegel repo?
Skärmavbild 2023-09-18 kl  13 59 37

File: src/app/components/popover-menu/popover-menu.component.html [hunk 2]
```diff
@@ -22,8 +22,7 @@
   >
     <tds-popover-menu-item>
       <a
-        target="_blank"
-        rel="noopener noreferrer"
+                rel="noopener noreferrer"
```
Comment (alexandereneroth) [inline src/app/components/popover-menu/popover-menu.component.html:25]: Why was target="_blank" removed? Just curious

File: src/app/components/toggle/toggle.component.ts [hunk 3]
```diff
@@ -37,17 +38,17 @@ import { Component, CUSTOM_ELEMENTS_SCHEMA, ElementRef, ViewChild } from '@angul
...
-  @ViewChild('toggle2', { static: true }) toggle2: ElementRef<HTMLTdsToggleElement> | undefined;
+  @ViewChild('toggle1', { static: true }) toggle1: HTMLTdsToggleElement;
+  @ViewChild('toggle2', { static: true }) toggle2: HTMLTdsToggleElement;
   handleTdsToggle(event: CustomEvent) {
     console.log('tdsToggle event received', event.detail);
   }
 
   toggleProgrammatically() {
-    this.toggle1?.nativeElement.toggle();
-    this.toggle2?.nativeElement.toggle();
```
Comment (alexandereneroth) [inline src/app/components/toggle/toggle.component.ts:51]: Nice, is this a benefit of having wrappers?

File: src/app/components/popover-menu/popover-menu.component.html (hunk 2 above)
Comment (alexandereneroth) [reply src/app/components/popover-menu/popover-menu.component.html:25]: ah doesnt really matter IMO

**Comment by alexandereneroth** [comment]
> > Getting this in web inspector console. Skärmavbild 2023-09-18 kl 10 39 58
> 
> Sweet, might need a null check in the actual component. So the fix might be in the @scania/tegel package

Actually some of the popovers on the start page are broken in this (they dont open) on the non wrapper page they all work, and theres no error message https://tegel-angular-demo.netlify.app/

**Comment by timrombergjakobsson** [comment]
> > > Getting this in web inspector console. Skärmavbild 2023-09-18 kl 10 39 58
> > 
> > 
> > Sweet, might need a null check in the actual component. So the fix might be in the @scania/tegel package
> 
> Actually some of the popovers on the start page are broken in this (they dont open) on the non wrapper page they all work, and theres no error message https://tegel-angular-demo.netlify.app/

Hm yeah, your right.

**Comment by alexandereneroth** [comment]
> > > > Getting this in web inspector console. Skärmavbild 2023-09-18 kl 10 39 58
> > > 
> > > 
> > > Sweet, might need a null check in the actual component. So the fix might be in the @scania/tegel package
> > 
> > 
> > Actually some of the popovers on the start page are broken in this (they dont open) on the non wrapper page they all work, and theres no error message https://tegel-angular-demo.netlify.app/
> 
> Hm yeah, your right.

Appears to be caused by angular wrapping the html element of the popover button in something. Investigating

**Comment by alexandereneroth** [comment]
> > > > > Getting this in web inspector console. Skärmavbild 2023-09-18 kl 10 39 58
> > > > 
> > > > 
> > > > Sweet, might need a null check in the actual component. So the fix might be in the @scania/tegel package
> > > 
> > > 
> > > Actually some of the popovers on the start page are broken in this (they dont open) on the non wrapper page they all work, and theres no error message https://tegel-angular-demo.netlify.app/
> > 
> > 
> > Hm yeah, your right.
> 
> Appears to be caused by Angular wrapping the html element of the popover button in something. Investigating.
> 
> Screenshot 2023-09-18 at 14 58 44

Stuck on this.

**Comment by TatyanaBumazhkina** [comment]
Added comments in Jira ticket regarding found issues

**Comment by TatyanaBumazhkina** [comment]
Angular page with wrapper works fine, passed

**Review by alexandereneroth — APPROVED** [review]

**Review by theJohnnyMe — APPROVED** [review]
Nice!

**Pull request merged**